src/onefetch/repo.rs

```diff
@@ -83,11 +83,12 @@ impl<'a> Repo<'a> {
 
         authors.truncate(n);
 
-        let authors: Vec<(String, usize, usize)> = authors
+        let authors: Vec<(String, Option<String>, usize, usize)> = authors
             .into_iter()
             .map(|(author_email, author_nbr_of_commits)| {
                 (
                     author_name_by_email.get(&author_email).unwrap().trim_matches('\'').to_string(),
+					if include_email { Some(author_email) } else { None },
```

Comment:
```suggestion
                    include_email.then(|| author_email),
```
See if this works. [`then`](https://doc.rust-lang.org/std/primitive.bool.html#method.then) is a bit more readable IMO
(although I would prefer the `then_some` which is currently nightly-only).
Please note that this is *not* a necessary change, because the if-else expression is also fine IMO.

However, this suggestion also changes tab indentation to spaces, which is a bit necessary to keep the code style
consistent.